docs/source/broadcasting.md

```diff
@@ -43,16 +43,17 @@ described above using the web player or ffplay.
 
 ## Playing the Stream
 
-You can request your stream in a number of ways.
-
-* Request the stream using your channel through the per-broadcaster [web player](http://media.livepeer.org). Use the Eth address that is printed out in the Livepeer CLI or the node output. 
-* Request the stream using the manifest ID through the [web player](http://media.livepeer.org/player.html).
+You can request your stream from your local node, or remotely if your node is exposed to the world.
+ 
 * Request the stream using `ffplay`
 
 ```
 $ ffplay http://localhost:8935/stream/{streamID}.m3u8
 ```
 
+* Request the stream using the manifest ID through the [web player](http://media.livepeer.org/player.html) which should pull from localhost only.
```

Comment:
I think @ya7ya merged a way to specify the host within the web player, but I'm not sure precisely how to do that.

Reply:
Yah I don't see it when I view the player.html page. So for now, ok with merging this, and can update accordingly if we have a more flexible page?

Reply:
The change was done to the "new" media.livepeer.org player, not the "old" player.html. @ya7ya can you describe how to specify the host in the new player?

In any case, if this updated information solves an immediate problem because folks are getting confused, I'm OK with merging.

Reply:
@j0sh @dob  , to specify a different host, use the url query `?source=`
reference docs: https://github.com/livepeer/livepeerjs/tree/master/packages/player#changing-stream-route-on-the-fly

Reply:
FWIW I couldn't get that working at media.livepeer.org - is the latest code deployed there? Going to go ahead with the merge, as I think asking the user to craft that URL is something we can include in the docs, but probably isn't the best initial experience anyway.